components/AuthorProfile.vue

```diff
@@ -80,22 +80,27 @@ const imageSizePx = ref(imageSize.value + 'px')
     </div>
     <div class="col-fixed profile">
       <div class="author-image">
-        <v-flexible-link :to="profileLink" raw>
+        <v-flexible-link
+          :to="profileLink"
+          raw
+          :aria-hidden="true"
```

Comment:
hmmm you think I should add these 2 lines when "raw" is added?

Reply:
no, that's unrelated to these changes